extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -539,27 +539,11 @@ export async function promptConnectionStringPasswordAndUpdateConnectionString(co
 			}
 		}
 		// set connection string to not include the password if connection info does not include password, or user chooses to not include password, or authentication type is not sql login
-		let userPassword: string | undefined;
 		if (includePassword !== constants.yesString) {
 			connectionString = await vscodeMssqlApi.getConnectionString(connectionDetails, false, false);
-
-			// Ask user to enter password if auth type is sql login and password is not saved
```

Comment:
So if a connection that already exists was chosen and doesn't have a password saved shouldn't we still be asking them whether they want to add the password in now? If I'm reading this right in that case we'll just skip over it and warn them that they need to enter it manually, but we should give them a choice to do it now if they want to save them time later. 

Can you confirm what happens currently with that scenario?

Reply:
Yes confirmed that it will skip over this entirely so I addressed this case as well. 
In the case where connection info does not have password, we will show the warning message but also show the prompt to enter.
<img width="684" alt="Screen Shot 2022-05-13 at 5 53 37 PM" src="https://user-images.githubusercontent.com/23587151/168404900-acda55ae-1686-477c-818e-ab802170d2f1.png">